ci/common/build.sh

```diff
@@ -46,10 +45,10 @@ build_deps() {
 }
 
 prepare_build() {
-  if [[ -n "${CLANG_SANITIZER}" ]]; then
+  if test -n "${CLANG_SANITIZER}" ; then
     CMAKE_FLAGS="${CMAKE_FLAGS} -DCLANG_${CLANG_SANITIZER}=ON"
   fi
-  if [[ "${BUILD_32BIT}" == ON ]]; then
+  if test "x${BUILD_32BIT}" = xON ; then
```

Comment:
Older implementations of `test` couldn't handle one side of the comparison being an empty string.  It likely isn't a problem on any current Linux, but may be on other platforms.  It may also help prevent misinterpreting the command if `$foo` resembles an operator (e.g., with `foo=-x`).